osu.Game.Rulesets.Catch.Tests/TestSceneCatcher.cs

```diff
@@ -23,7 +24,7 @@ public class TestSceneCatcher : CatchSkinnableTestScene
         [BackgroundDependencyLoader]
         private void load()
         {
-            SetContents(() => new Catcher
+            SetContents(() => new Catcher(new Container())
```

Comment:
as mentioned in discord, this container should be displayed in the test scene. you've regressed the test as it stands.

Reply:
This test used to provide `null` for `additiveTarget` so I thought there won't be a problem with providing a dummy `Container`? 

Moreover, providing a real `additiveTarget` here sounds exactly like what `CatcherArea` provides, which has an own test scene (`TestSceneCatcherArea`), why not just remove this test scene if it's worthless.

Reply:
i guess it's fine for this case, as input is also not handled here.